test-project/Assets/EditmodeTests/Conversions/CoordinatesTests.cs

```diff
@@ -0,0 +1,31 @@
+using Improbable;
+using NUnit.Framework;
+using UnityEngine;
+
+namespace EditmodeTests.Conversions
+{
+    public class CoordinatesTests
```

Comment:
All of these compare floats to doubles, after converting from double to float. I think these need an epsilon to compare with.

Reply:
Same with the value's we pass to constructors and operators, these should all be explicitly written as the right type (floats)